internal/precache/precache.go

```diff
@@ -100,61 +106,61 @@ type Status struct {
 }
 
 // CreateJob creates a new precache job.
-func CreateJob(ctx context.Context, c client.Client, config *Config) error {
+func (handler *PHandler) CreateJob(ctx context.Context, config *Config) error {
```

Comment:
maybe lets call handler -> h
Will make lines shorter. WDYT?

Reply:
Good suggestion @tsorya, will do that!